packages/hooks/src/useAcceptOffer.ts

```diff
@@ -73,7 +73,7 @@ export default function useAcceptOffer(signer: Signer | undefined): [
   },
 ] {
   const { sdk } = useContext(LiteflowContext)
-  const [{ pollOwnership, checkOwnership }] = useCheckOwnership()
+  const { pollOwnership, checkOwnership } = useCheckOwnership()
```

Comment:
Changed the API of `useCheckOwnership`. This is not a breaking change as `useCheckOwnership` is not exported.